.github/ISSUE_TEMPLATE/DOCUMENTATION.yaml

```diff
@@ -0,0 +1,44 @@
+name: Documentation
+description: Report typos/bugs or suggest new content.
+labels: ["documentation"]
+assignees: 'abbycross, beckykd, javabster'
+
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for reporting!  Please also use the search to see if there are any other relevant issues or pull requests.
+
+  - type: textarea
+    attributes:
+      label: URL to the relevant documentation
+      description: provide a URL and section name/paragraph if applicable
+    validations:
+      required: false
+
+  - type: checkboxes
+    attributes:
+      label: Select all that apply
+      description: Please select all that apply from the following.
+      options:
+        - label: typo
+        - label: code bug
+        - label: add new content
+        - label: out-of-date content
+        - label: it's just wrong
+        - label: other
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: What is the documentation issue/request?
+      description: If reporting a bug, please be as specific as possible (steps to reproduce, a screenshot etc.) If suggesting new content, please give a clear and concise description. What value does the new content bring?
```

Comment:
If you want to be able to split a long string into multiple source lines but have single newlines collapsed into spaces, the syntax is
```yaml
description: >
  If reporting a bug, please be as specific as possible ...
  If suggesting new content, ...
```
(but the big long string you've got here is fine too)

Reply:
So I just add the > and the rest takes care of itself? Or do I need the ellipses?

Reply:
Just the > on the top line - the ellipses were me being lazy about typing thing out sorry!

Reply:
I just wanted to be absolutely sure :) Thanks!